packages/gatsby/src/utils/page-ssr-module/bundle-webpack.ts

```diff
@@ -222,19 +222,32 @@ export async function createPageSSRBundle({
     ].filter(Boolean) as Array<webpack.WebpackPluginInstance>,
   })
 
+  let IMAGE_CDN_URL_GENERATOR_MODULE_RELATIVE_PATH = ``
+  if (global.__GATSBY?.imageCDNUrlGeneratorModulePath) {
+    await fs.copyFile(
+      global.__GATSBY.imageCDNUrlGeneratorModulePath,
+      path.join(outputDir, `image-cdn-url-generator.js`)
+    )
+    IMAGE_CDN_URL_GENERATOR_MODULE_RELATIVE_PATH = `./image-cdn-url-generator.js`
+  }
+
   let functionCode = await fs.readFile(
     path.join(__dirname, `lambda.js`),
     `utf-8`
   )
 
   functionCode = functionCode
-    .replace(
+    .replaceAll(
```

Comment:
this is not needed here, just was needed for `%IMAGE_CDN_URL_GENERATOR_MODULE_RELATIVE_PATH%` because it's used multiple times in the template and figured I would just adjust previous replacements as well to avoid possible future issues (like one I hit with new replacement)